status.py

```diff
@@ -0,0 +1,56 @@
+import datetime
+import json
+import requests
+
+from os import path
+from typing import Any, Dict
+
+from constants import base_payload, released_premined_tokens, server, total_premined
+
+
+def run(event: Dict[str, Any], context: 'bootstrap.LambdaContext') -> Dict[str, Any]:
+    """ Get mined tokens from the full node and return the economics status of the network
+    """
+    # Get mined tokens from full node
+    response = requests.get(path.join(server, 'mined_tokens'))
+    if response and response.status_code == 200 and response.json().get('mined_tokens'):
+        data = response.json()
+
+        mined_tokens = data['mined_tokens']
+        circulating_supply = mined_tokens + released_premined_tokens
+        total_supply = mined_tokens + total_premined
+
+        body = {
+            'latest_update': int(datetime.datetime.now().timestamp()),
+            'backend': server,
+            'total_supply': total_supply,
+            'circulating_supply': circulating_supply,
+            'mined_tokens': mined_tokens,
+            'released_premined_tokens': released_premined_tokens
+        }
+    else:
+        # XXX Uncoment it when the full node starts responding properly
```

Comment:
You need to uncomment these lines and remove the code below them.

Reply:
I will when the full node is updated with reply the mined tokens and I finish the test of this code 100%

Reply:
The code was updated now that we have the full node updated. Can you check again, please?